### PR TITLE
BETA: Register skythegmer.is-a.dev

### PR DESCRIPTION
**Pull request by skyy-codes**
Added `skythegmer.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by skyy-codes** [comment]
(google.com cname for now, i'll change that later on)

File: domains/skythegmer.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "skyy-codes",
+        "email": "skyxgamiing@gmail.com"
+    },
+    "record": {
+        "CNAME": "google.com"
```
Comment (CuteDog5695) [inline domains/skythegmer.json:7]: please replace google.com with your site

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/skythegmer.json (hunk 1 above)
Comment (skyy-codes) [reply domains/skythegmer.json:7]: I'll do that in a sec

**Comment by TerraPlayz** [comment]
Any Updates??

**Pull request closed without merging**